flytekit/clients/auth/token_client.py

```diff
@@ -31,15 +31,13 @@ class DeviceCodeResponse:
     {'device_code': 'code',
          'user_code': 'BNDJJFXL',
          'verification_uri': 'url',
-         'verification_uri_complete': 'url',
```

Comment:
this is okay to take out.

Reply:
wdym. to revert this?

Reply:
nah i mean it's okay to take it out of the test.  don't revert this.